core/src/Instance/B2CAuthority.cs

```diff
@@ -50,8 +55,15 @@ internal B2CAuthority(string authority, bool validateAuthority)
 
             CanonicalAuthority = string.Format(CultureInfo.InvariantCulture, B2CCanonicalAuthorityTemplate, authorityUri.Authority,
                 pathSegments[0], pathSegments[1], pathSegments[2]);
+        }
 
-            AuthorityType = AuthorityType.B2C;
+        protected override string GetDefaultOpenIdConfigurationEndpoint()
+        {
+            Uri b2cAuthority = new Uri(CanonicalAuthority);
+
+            // Ignoring sovereign clouds
```

Comment:
>// Ignoring sovereign clouds [](start = 12, length = 28)

to support host we would need to make the call to the tenant discovery end point, which is not supported for b2clogin.com